index.html

```diff
@@ -0,0 +1,490 @@
+                                        <div class="user-comment-replies">
+                                            <i data-feather="chevron-down" width="15"></i> <span class="ms-2 mt-5">3 REPLIES</span>
+                                        </div>
+                                    </div>
+                                </div>
+                                <div class="ind-user-comment d-flex justify-content-start mt-4">
+                                    <div class="commenter-image col-1">
+                                        <img src="img/trey-songz.jpg" alt="Trey Songz" class="rounded-circle m-auto" style="width: 40px; height: 40px;">
+                                    </div>
+                                    <div class="form-box col-11">
+                                        <div class="user-info">
+                                            <p style="font-size: 15px;">John Richard <span style="color: #a0a0a0; font-size: 12px;">1 year ago</span></p>
+                                        </div>
+                                        <div class="user-comment-text">
+                                            <p>To be honest it's been months since I started forex , finding it difficult to stand out , someone will say be patient maybe it's really what I lack or something , watching your videos as been great but am not just  the impact ,I could go back to my chart and still make a lot of losses</p>
+                                        </div>
+                                        <div class="user-comment-likes d-flex justify-content-start">
+                                            <i data-feather="thumbs-up" width="15"></i> <span class="ms-2 mt-1">15</span>
+                                            <i data-feather="thumbs-up" width="15" class="ms-4"></i> <span class="ms-2 mt-1">3</span>
+                                            <p class="ms-4 mt-1">REPLY</p>
+                                        </div>
+                                        <div class="user-comment-replies">
+                                            <i data-feather="chevron-down" width="15"></i> <span class="ms-2 mt-5">3 REPLIES</span>
+                                        </div>
+                                    </div>
+                                </div>
+                                <div class="ind-user-comment d-flex justify-content-start mt-4">
+                                    <div class="commenter-image col-1">
+                                        <img src="img/trey-songz.jpg" alt="Trey Songz" class="rounded-circle m-auto" style="width: 40px; height: 40px;">
+                                    </div>
+                                    <div class="form-box col-11">
+                                        <div class="user-info">
+                                            <p style="font-size: 15px;">John Richard <span style="color: #a0a0a0; font-size: 12px;">1 year ago</span></p>
```

Comment:
Please do not use inline styles. Keep your style definitions in the external CSS file you created.